include/iotc_types.h

```diff
@@ -204,33 +188,31 @@ typedef union iotc_crypto_key_union_u {
 } iotc_crypto_key_union_t;
 
 /**
- * @enum iotc_crypto_key_signature_algorithm_e
- * @brief specifies a key signature algorithm. This may be used to
- * signify a key type, or to specify how a JWT might be signed.
- * This should match the algorithm type that was used to provision the device's
- * public key in Google IoT Core.
+ * @enum  iotc_crypto_key_signature_algorithm_e
+ * @brief Define a key signature algorithm to sign JWTs.
  *
- * The value provided here will also determine the "alg" field of the JWT, which
- * is automatically assembled by the IoTC Client during the connection process.
+ * This value also determines the JWT "alg" field, which the client application
+ * automatically assembles when it connects to Cloud IoT Core. Cite the same
+ * algorithm in <a href="https://cloud.google.com/iot/docs/how-tos/devices#creating_device_key_pairs">Cloud IoT Core</a>.
  *
- * IOTC_CRYPTO_KEY_SIGNATURE_ALGORITHM_INVALID - for internal use only.
- * IOTC_CRYPTO_KEY_SIGNATURE_ALGORITHM_ES256 - to denote ECDSA using P-256
- * and SHA-256
+ * <b>Note</b>: RSASSA-PKCS1-v1_5 with SHA-256 (RS256) is not supported.
  *
- * NOTE: RSASSA-PKCS1-v1_5 using SHA-256 (aka RS256) is currently not supported.
+ * @retval IOTC_CRYPTO_KEY_SIGNATURE_ALGORITHM_INVALID for development.
+ * @retval IOTC_CRYPTO_KEY_SIGNATURE_ALGORITHM_ES256 ECDSA with P-256 and
+ *     SHA-256.
  */
 typedef enum iotc_crypto_key_signature_algorithm_e {
   IOTC_CRYPTO_KEY_SIGNATURE_ALGORITHM_INVALID = 0,
   IOTC_CRYPTO_KEY_SIGNATURE_ALGORITHM_ES256
 } iotc_crypto_key_signature_algorithm_t;
 
 /* @struct iotc_crypto_key_data_t
- * @brief A structure that contains a iotc_crypto_key_union, an
- * enumeration of which configuration of the union that's being used,
- * and an enumeration of the signature algorithm of the key.
+ * @brief Set iotc_crypto_key_union.
```

Comment:
Set?  This is a union structure. Set implies a verb, or an action. This is a noun.